src/mcts/params.cc

```diff
@@ -192,6 +195,7 @@ void SearchParams::Populate(OptionsParser* options) {
   // Many of them are overridden with training specific values in tournament.cc.
   options->Add<IntOption>(kMiniBatchSizeId, 1, 1024) = 256;
   options->Add<IntOption>(kMaxPrefetchBatchId, 0, 1024) = 32;
+  options->Add<BoolOption>(kLogitQEnabledId) = true;
```

Comment:
This should be overridden to false in selfplay.

Reply:
Somewhere in `/src/selfplay`?

Reply:
I would guess what line 195 says (`tournament.cc`).